### PR TITLE
Remove redundant file creation message, and always overwirte files when converting

### LINKED ISSUE #18: Redundant file creation message
```console
$ ../kompose convert 
file abc.yaml has been created
file abc.yaml has been created
file abc.yaml has been created
file abc.yaml has been created
```

I'll send a PR for this.

### PR DESCRIPTION
**Pull request by janetkuo**
Fixes #18

cc @runseb @ngtuna

### CODE REVIEW

File: cli/app/app.go [hunk 1]
```diff
@@ -38,46 +38,47 @@ import (
...
+	"Tty":           "",
+	"User":          "",
+	"VolumeDriver":  "",
+	"VolumesFrom":   "",
+	"Expose":        "",
 	"ExternalLinks": "",
-	"LogOpt": "",
-	"ExtraHosts": "",
+	"LogOpt":        "",
+	"ExtraHosts":    "",
```
Comment (janetkuo) [inline cli/app/app.go]: Changes above this line are just `gofmt`

**Comment by ngtuna** [comment]
I found out somehow `os.Create()` causes this below issue:

```
$ kompose convert -o abc.txt -f docker-gitlab.yml
FATA[0000] Failed to write svc to file: write abc.txt: bad file descriptor 
```

**Comment by janetkuo** [comment]
@ngtuna no, it's because the file is closed early, see the fix #24

**Comment by ngtuna** [comment]
Ah my bad. Thanks @janetkuo

**Pull request merged**